src/ReflectionProperty.php

```diff
@@ -113,11 +113,31 @@ public function __debugInfo(): array
      */
     public function __toString()
     {
+        $defaultValueDisplay = '';
+
+        if ($this->isDefault()) {
+            $this->__initialize();
+            $defaultValue = $this->getDefaultValue();
+            if (is_string($defaultValue)) {
+                if (strlen($defaultValue, 18)) {
+                    $defaultValue = substr($defaultValue, 0, 15) . '...';
+                }
+
+                $defaultValue = "'" . $defaultValue . "'";
+            }
+
+            if ($defaultValue === null) {
+                $defaultValue = "NULL";
+            }
+
+            $defaultValueDisplay = '= ' . $defaultValue;
+        }
```

Comment:
@lisachenko this is for display property with default value display

Reply:
To be honest, this crazy export/_toString logic for some Reflection classes might be dropped - it is not needed directly by goaop/framework. Mostly, this was implemented for compatibility previously

Reply:
I am thinking that keeping map with native API is the first step, we can remove when everything compatible 👍

Reply:
Yes, this might be easier to achieve, indeed. 

Just wanted to highlight that these two methods can generate some troubles, but not used directly by the goaop/framework and we might just drop support for this method.